tests/cassette/keep_alive_monitor_remote_ping.yaml

```diff
@@ -0,0 +1,66 @@
+interactions:
```

Comment:
Could you add to the PR description what was the process to generate this yaml file just for later reference?

Reply:
I added notes to CONTRIBUTING.md on how API testing can be done with VCR.py, that uses the keepalive cassette as the example. Hopefully this helps in also converting notifier tests to hermetic. The downside is that then to truly run live, one needs to kill the matching cassette(s). We could allow re-recording them without the delete, but then the risk of someone accidentally committing their tokens is a bit too high.